ts/features/bonus/bpd/screens/details/transaction/detail/BpdTransactionDetailComponent.tsx

```diff
@@ -32,6 +33,9 @@ const styles = StyleSheet.create({
     borderWidth: 1,
     top: -10,
     height: 25
+  },
+  copyText: {
+    width: widthPercentageToDP("70%")
```

Comment:
```suggestion
    flex: 1,
    paddingRight: 16
```
In order to not to be dependent on the size of the screen and possibly make the component reusable when it is not full screen. What you think?

Reply:
surely better!